rest/data/interface.go

```diff
@@ -399,3 +399,17 @@ type Connector interface {
 	// CompareTasks returns the order that the given tasks would be scheduled, along with the scheduling logic.
 	CompareTasks([]string, bool) ([]string, map[string]map[string]string, error)
 }
+
+type MockConnector interface {
+	// Commit queue methods
+	// GetGithubPR takes the owner, repo, and PR number.
+	GetGitHubPR(context.Context, string, string, int) (*github.PullRequest, error)
+	// if bool is true, move the commit queue item to be processed next.
+	EnqueueItem(string, restModel.APICommitQueueItem, bool) (int, error)
```

Comment:
I don't believe this function needs to be mocked; it just adds something to the commit queue

Reply:
You're right I was able to remove it with a bit of refactoring